src/canister_tests/Cargo.toml

```diff
@@ -3,7 +3,7 @@ name = "canister_tests"
 version = "0.1.0"
 edition = "2021"
 
-[dev-dependencies]
+[dependencies]
```

Comment:
why change this? do we expect anything to be built except for the tests? or is that necessary for crates that depend on this?

Reply:
Yes, this is required because `#[cfg(test)]` does not propagate to dependencies.